tests/test_utils_torch.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
+
+"""
+Unit tests for parlai.utils.torch.
+"""
+
+import torch
+import numpy as np
+import unittest
+from parlai.utils.torch import padded_tensor, argsort, PipelineHelper, neginf
+
+
+class TestTorchUtils(unittest.TestCase):
+    def test_neginf(self):
+        assert neginf(torch.float32) < -1e15
+        assert neginf(torch.float16) > -1e15
+        assert neginf(torch.float16) < -1e4
+
+    def test_padded_tensor(self):
+        # list of lists
+        lol = [[1, 2], [3, 4, 5]]
```

Comment:
ha nice

Reply:
lol that was just moving this function around.